src/BibleWell.App/Views/Components/TiptapRendererView.axaml.cs

```diff
@@ -7,19 +7,23 @@
 using Avalonia.Media;
 using BibleWell.App.ViewModels.Components;
 using BibleWell.Aquifer;
+using CommunityToolkit.Mvvm.DependencyInjection;
+using Microsoft.Extensions.Logging;
 
 namespace BibleWell.App.Views.Components;
 
 // ReSharper disable All
 public partial class TiptapRendererView : UserControl
 {
     private readonly Grid? _container;
+    private readonly ILogger<TiptapRendererView> _logger;
 
     public TiptapRendererView()
     {
         DataContextChanged += OnDataContextChanged;
         InitializeComponent();
         _container = ContentContainer;
+        _logger  = Ioc.Default.GetRequiredService<ILogger<TiptapRendererView>>();
```

Comment:
👍 I look into why constructor dependency injection isn't working later on but this will work for now.